apps/mobile/src/components/Post/PostInput.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from 'clsx';
+import { useColorScheme } from 'nativewind';
+import { useState } from 'react';
+import { TextInput, View } from 'react-native';
+
+import colors from '~/shared/theme/colors';
+
+import { Typography } from '../Typography';
+
+export function PostInput() {
+  const [value, setValue] = useState('');
+
+  const { colorScheme } = useColorScheme();
+
+  const isTextTooLong = value.length >= 600;
+  const characterCount = value.length;
+
+  const handleChangeText = (newText: string) => {
+    if (newText.length <= 600) {
+      setValue(newText);
+    }
+  };
+
+  return (
+    <View
+      className="relative border bg-offWhite dark:bg-black-900 border-porcelain dark:border-black-500 pb-10"
+      style={{
+        height: 117,
+      }}
+    >
+      <TextInput
```

Comment:
oo could you make the text input auto-focus when it's mounted? so that the keyboard is open and ready to go

Reply:
bump @jakzaizzat

Reply:
oo also noticed a potential bug if the keyboard is open:
if I tap back or "x" while I'm typing, the keyboard remains open so I can't see the confirmation bottom sheet. 
I think we should unfocus the keyboard . not 100% if this is the same behavior on device but I think so


https://github.com/gallery-so/gallery/assets/80802871/81534438-36bc-41f2-a2a9-0eff9f736e35